### ISSUE TITLE
Incorporate LanguageDsc Module into ComputerManagementDsc

### ISSUE THREAD

**Issue by PlagueHO** [PowerShell/xComputerManagement#127]
@ld0614 has proposed merging his [LanguageDsc](https://github.com/ld0614/LanguageDsc) module into this one instead of SystemLocaleDsc (which can then be deprecated). This will be done shortly after completing issue #119.

This issue is a placeholder to ensure clarity of the plan is available to the community. See https://github.com/PowerShell/DscResources/issues/288 for more information.

**Comment by PlagueHO** [comment]
@ld0614 - I'm about to begin the renaming of this resource module over the ComputerManagementDsc, so are you still happy to contribute your module once that one is through?

**Comment by ld0614** [comment]
Yep, once the module has been renamed I'll submit an initial PR.  I'm expecting a number of improvements will be required to bring it up to fully HQRM compliant but I'm eager to kick the review process off 😄
